yt_dlp/extractor/loco.py

```diff
@@ -0,0 +1,82 @@
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class LocoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?loco\.com/(?P<type>streamers|stream)/(?P<id>[^/?#]+)'
+    _TESTS = [{
+        'url': 'https://loco.com/streamers/teuzinfps',
+        'info_dict': {
+            'id': 'teuzinfps',
+            'ext': 'mp4',
+            'title': r're:MS BOLADAO, RESENHA & GAMEPLAY ALTO NIVEL',
+            'description': 'bom e novo',
+            'uploader_id': 'RLUVE3S9JU',
+            'channel': 'teuzinfps',
+            'channel_follower_count': int,
+            'comment_count': int,
+            'view_count': int,
+            'concurrent_view_count': int,
+            'like_count': int,
+            'thumbnail': 'https://static.ivory.getloconow.com/default_thumb/743701a9-98ca-41ae-9a8b-70bd5da070ad.jpg',
+            'tags': ['MMORPG', 'Gameplay'],
+            'series': 'Tibia',
+            'timestamp': int,
```

Comment:
Does the test harness look for `upload_date` too when `timestamp` is set, as in upstream? Or even `modified_date`? But if the test doesn't complain, that's fine.

Reply:
Tests pass fine but I don't quite understand what you mean precisely